src/navigation-bar.css.ts

```diff
@@ -179,7 +179,7 @@ export const navigationBarContentRight = style({
     display: 'flex',
     flex: 1,
     alignItems: 'center',
-    justifyContent: 'end',
+    justifyContent: 'flex-end',
```

Comment:
Unrelated to this task. This was causing a warning from Vite when executing `yarn build`